pkg/broker/rbac.go

```diff
@@ -54,7 +54,7 @@ func NewBrokerAdminRole() *rbacv1.Role {
 			rbacv1.PolicyRule{
 				Verbs:     []string{"create", "get", "list", "watch", "patch", "update", "delete"},
 				APIGroups: []string{"submariner.io"},
-				Resources: []string{"clusters", "endpoints"},
+				Resources: []string{"clusters", "endpoints", "gateways"},
```

Comment:
Those rbacs are for the broker cluster, we should not handle gateways on the broker cluster. This admin role is for subctl over the broker.